tests/unit/Model/SQS/MarketingPreferences.model.js

```diff
@@ -168,8 +168,7 @@ describe('Model/MarketingPreferencesModel', () => {
       transSource: 'giftaid-sportrelief',
       transSourceUrl: 'https://giftaid.sportrelief.com/',
       transType: 'prefs',
-      email: '',
```

Comment:
from title : 'Ensure validation fails when email permission is set and no email is provided'
I think `email` should be present, and `permissionEmail` should be set to 1, right ?

Reply:
No, we want validation to fail when no email is provided but the permission is set to `1` or `0`